docs/modules/ROOT/pages/includes/quarkus-openapi-generator-moqu-wiremock.adoc

```diff
@@ -15,7 +15,7 @@ endif::add-copy-button-to-config-props[]
 
 [.description]
 --
-Path to the Moqu (relative to the project).
+Path to the Moqu OpenAPI files, relative to the `src/main/resources` directory.
```

Comment:
@mcruzdev when sending PRs to server/moqu make sure to run the whole build, from the top, locally.